build/lib/release.sh

```diff
@@ -214,10 +214,40 @@ function kube::release::package_node_tarballs() {
   done
 }
 
+function kube::release::is_current_builder_missing_target_arch() {
+  local current_builder
+  current_builder="$(docker buildx inspect)"
+  if grep -q "^Driver: docker$"  <<<"${current_builder}"; then
+    return 0
+  fi
+  local platform
+  for platform in "${KUBE_SERVER_PLATFORMS[@]}"; do
+    if ! grep -q "${platform}" <<<"${current_builder}"; then
+      return 0
+    fi
+  done
+  return 1
+}
+
 # Package up all of the server binaries in docker images
 function kube::release::build_server_images() {
   # Clean out any old images
   rm -rf "${RELEASE_IMAGES}"
+
+  if kube::release::is_current_builder_missing_target_arch; then
+    local sudo=""
+    if [[ $(id -u) != 0 ]]; then
+        sudo=sudo
+    fi
+
+    ${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset
+
+    docker buildx rm kube-multiarch-builder || true
```

Comment:
this one doesn't match the builder names below.

we should probably dry this out into a variable.

Reply:
Done.